src/util/option-utils.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+var ConfigService = require('../service/configuration-service');
+
+var urlConfig = new ConfigService().get('server');
+var customDefaults = {};
+var libDefaults = {
+    /**
+     * The account id. In the Epicenter UI, this is the **Team ID** (for team projects) or **User ID** (for personal projects). Defaults to empty string. If left undefined, taken from the URL.
+     * @type {String}
+     */
+    account: urlConfig.accountPath,
+    /**
+     * The account id. In the Epicenter UI, this is the **Team ID** (for team projects) or **User ID** (for personal projects). Defaults to empty string. If left undefined, taken from the URL.
+     * @type {String}
+     */
+    project: urlConfig.projectPath,
+    isLocal: urlConfig.isLocalhost,
+    store: {}
+};
+
+var optionUtils = {
+    /**
+     * Gets the final options by overriding the global options set with
+     * optionUtils#setDefaults() and the lib defaults.
+     * @param {object} `options` The final options object.
+     */
+    getOptions: function (options) {
```

Comment:
Nitpick on the syntax - `getOptions(options)` seems a little black-magicy where I can't tell what I'm getting back. Maybe `getMergedOptions`?